plugins/services/src/js/structs/Application.js

```diff
@@ -230,4 +230,8 @@ module.exports = class Application extends Service {
 
     return `${Config.rootUrl}/service/${serviceName}/`;
   }
+
+  isFolder() {
```

Comment:
ℹ️  If you ever want to add method to the _service interface_ please make sure to add it to the base `Service` struct and `ServiceTree` as all "services" should share the same interface.